app/jobs/fetch_response_map_job.rb

```diff
@@ -0,0 +1,24 @@
+class FetchResponseMapJob < ApplicationJob
+  queue_as :default
+  require 'uri'
+	require 'net/http'
+
+  def perform(consultation_response)
+  	cr = consultation_response
+  	url = "http://35.223.43.106/venter/modelKM"
```

Comment:
🚨 @anondita civis is open sourced. Don't expose the IP here. Also squash this commit please. 🚨